docs/docs/EventsApi.md

```diff
@@ -331,7 +331,7 @@ Name | Type | Description  | Notes
  **itemsPerPage** | **int** | Number of items that the response returns per page. | [default to 100]
  **pageNum** | **int** | Number of the page that displays the current set of the total objects that the response returns. | [default to 1]
  **clusterNames** | **[]string** | Human-readable label that identifies the cluster. | 
- **eventType** | **[]string** | Category of incident recorded at this moment in time.  **IMPORTANT**: The complete list of event type values changes frequently. | 
+ **eventType** | **[][]string** | Category of incident recorded at this moment in time.  **IMPORTANT**: The complete list of event type values changes frequently. | 
```

Comment:
Array of array?

Reply:
Yes. It looks like issue. Investigating that.

Reply:
Fixed. It was basically leftover transformation connected with unusual solution for the problem upstream.